src/videos.js

```diff
@@ -0,0 +1,124 @@
+    },{
+        id: 11,
+        title: 'Video 11',
+        descripcion: 'Información sobre video 11',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 12,
+        title: 'Video 12',
+        descripcion: 'Información sobre video 12',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 13,
+        title: 'Video 13',
+        descripcion: 'Información sobre video 13',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 14,
+        title: 'Video 14',
+        descripcion: 'Información sobre video 14',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'    
+    },{
+        id: 15,
+        title: 'Video 15',
+        descripcion: 'Información sobre video 15',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 16,
+        title: 'Video 16',
+        descripcion: 'Información sobre video 16',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 17,
+        title: 'Video 17',
+        descripcion: 'Información sobre video 17',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 18,
+        title: 'Video 18',
+        descripcion: 'Información sobre video 18',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 19,
+        title: 'Video 19',
+        descripcion: 'Información sobre video 19',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 20,
+        title: 'Video 21',
+        descripcion: 'Información sobre video 21',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 22,
+        title: 'Video 22',
+        descripcion: 'Información sobre video 23',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 24,
+        title: 'Video 24',
+        descripcion: 'Información sobre video 24',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    },{
+        id: 25,
+        title: 'Video 25',
+        descripcion: 'Información sobre video 25',
+        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+    }
+
+]
+
+export default videos;
```

Comment:
I believe that there was a mock data file in the base project, you can use it without problem if you want, if not, it's ok to have this file